config/routes.rb

```diff
@@ -28,6 +28,10 @@
       end
 
       resources :versions, only: [:index, :show], format: :json
+      resources :versions, only: [:show] do
+        get 'raw', on: :member
+      end
+      resources :raw, only: [:show]
```

Comment:
This is setting up a route to the `show` method on the controller at `app/controllers/api/v0/raw`, which doesn’t exist in this PR. Is that something you were planning to add?

Reply:
same as above. will cleanup.